xbmc/GUIPassword.h

```diff
@@ -25,8 +25,21 @@ class CGUIPassword : public ISettingCallback
 public:
   CGUIPassword(void);
   ~CGUIPassword(void) override;
+  template <typename T>
+  bool IsItemUnlocked(T pItem, const std::string& strType, const std::string& strLabel,
+                       const std::string& strHeading);
   bool IsItemUnlocked(CFileItem* pItem, const std::string &strType);
+  /* \brief Tests if the user is allowed to access the share folder
```

Comment:
Please move the doxy before the definition like most of the codebase.

Reply:
done